packages/ui-api/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@polkadot/ui-api",
+  "version": "0.1.0",
+  "repository": "https://github.com/polkadot-js/light-ui.git",
+  "license": "Apache-2.0",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "yarn build --watch",
+    "test": "echo skipped"
+  },
+  "dependencies": {
+    "@polkadot/api": "^0.42.2",
+    "@polkadot/types": "^0.33.25",
+    "@polkadot/ui-keyring": "^0.23.2",
```

Comment:
note: latest is "@polkadot/ui-keyring": "^0.25.9"

Reply:
Looking at the above, types are lagging behind API, should be aligned. (We could possibly not even add it, it is a deep of API)